scripts/flattener.py

```diff
@@ -151,7 +180,7 @@
 
 EPILOG = '''
 Examples:
-
+HsapDv:0000264
```

Comment:
I'm not understanding why this is here

Reply:
Oh dear, that should not be there. Cut and paste error. I'll get all the fixes in once you've had a chance to review the remainder of the code.